src/main/resources/META-INF/hudson.remoting.ClassFilter

```diff
@@ -0,0 +1,10 @@
+# For JENKINS-49574
+# Calendar class implementation has a custom deserialization logic, but it seems to be safe.
+java.util.Calendar
```

Comment:
Kill. IIUC all this is just for `CVSChangeLog.changeDate`, right? So switch that to a `long` and be done with it.